frontend/src/components/Board/Board.jsx

```diff
@@ -1,34 +1,18 @@
-import {
-  BoardInfoContainer,
-  BoardInfo,
-  BoardImage,
-  BoardTitle,
-  AuthorInfoContainer,
-  AuthorName,
-  AuthorAvatar,
-} from './Board.styled';
-import DotsButton from 'components/DotsButton/DotsButton';
-import Dropdown from 'components/Dropdown/Dropdown';
-import { FaUserCircle } from 'react-icons/fa';
+import { BoardContainer } from './Board.styled';
+import BoardTitleSection from './BoardTitleSection/BoardTitleSection';
+import BoardDescription from './BoardDescription/BoardDescription';
```

Comment:
Komponenty powinny być w folderze components, a nie components/Board

Reply:
ale że jak
<img width="271" alt="Zrzut ekranu 2022-02-6 o 13 57 34" src="https://user-images.githubusercontent.com/27691628/152681881-ead53ce6-4faf-4d33-a231-50f0177ddb42.png">
przecież jak wyjme wszystko z /Board to bedzie luzem lezało w components

Reply:
tak miało być według tego co mówił Piotrek. Powinny być foldery components/Dropdown, components/DotsButton, components/BoardTitleSection itd.

Reply:
a w ten sposob okej rozumiem, myslalem ze chcesz zeby to wszystko wyjąć z folderu Board xD
okej moge tak zrobic; moj zamysl byl taki, ze tak bedzie bardziej uporządkowane